src/constraint_system/standard/quotient_poly.rs

```diff
@@ -156,7 +159,9 @@ fn compute_circuit_satisfiability_equation(
             let b_4 = delta(w4_next - four * wl);
             let b = (b_1 + b_2 + b_3 + b_4) * qrange;
 
-            (a + b) * v_h_i
+            let c = qlogic * ((wl - wr) * wo);
```

Comment:
The check for the logic gates cannot be just `(wl - wr) * wo)`

Reply:
Agree as we discussed.

Actually missing:
- XOR/AND constraint
- Base4 (quad) constraint (For the input wires and output)